applications/FluidDynamicsApplication/custom_processes/compute_pressure_coefficient_process.h

```diff
@@ -0,0 +1,199 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Eduard Gómez
+//
+//
+
+#ifndef KRATOS_COMUTE_PRESSURE_COEFFICIENT_PROCESS_H
+#define KRATOS_COMUTE_PRESSURE_COEFFICIENT_PROCESS_H
+
+// System includes
+#include <string>
+#include <iostream>
+
+// External includes
+
+// Project includes
+#include "includes/model_part.h"
+#include "includes/define.h"
+#include "processes/process.h"
+#include "includes/kratos_parameters.h"
+
+// Application includes
+
+
+namespace Kratos
+{
+///@addtogroup FluidDynamicsApplication
+///@{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+using NodeType = ModelPart::NodeType;
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/// Utility to modify the distances of an embedded object in order to avoid bad intersections
+/// Besides, it also deactivate the full negative distance elements
+class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ComputePressureCoefficientProcess : public Process
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of ComputePressureCoefficientProcess
+    KRATOS_CLASS_POINTER_DEFINITION(ComputePressureCoefficientProcess);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Constructor with Kratos parameters.
+    ComputePressureCoefficientProcess(
+        ModelPart& rModelPart,
+        Parameters Params);
```

Comment:
If not strictly required, I'd remove the old model part based constructor.

Reply:
Agreed